<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Airbus Fly-By-Wire Development</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabConnection">
       <attribute name="title">
        <string>Connection</string>
       </attribute>
       <widget class="QPushButton" name="pushButtonConnect">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>CONNECT</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxConfigurationIndex">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>10</y>
          <width>42</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxPitch">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>10</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Pitch Input Masked</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxRoll">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>30</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Roll Input Masked</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabPitch">
       <attribute name="title">
        <string>Pitch</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelPitchKp">
            <property name="text">
             <string>Load Demand Kp</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBoxPitchLoadDemandKp">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPitchLoadDemandKi">
            <property name="text">
             <string>Load Demand Ki</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBoxPitchLoadDemandKi">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPitchLoadDemandKd">
            <property name="text">
             <string>Load Demand Kd</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBoxPitchLoadDemandKd">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonPitchUpdate">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelPitchRateKp">
            <property name="text">
             <string>C* Kp</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBoxPitchRateKp">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.200000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPitchRateKi">
            <property name="text">
             <string>C* Ki</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBoxPitchRateKi">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.400000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPitchRateKd">
            <property name="text">
             <string>C* Kd</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBoxPitchRateKd">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.005000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QSlider" name="sliderGainPitch">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelPitch"/>
        </item>
        <item row="4" column="0">
         <widget class="ActualDemandChart" name="loadFactor"/>
        </item>
        <item row="5" column="0">
         <widget class="ActualDemandChart" name="pitchRate"/>
        </item>
        <item row="6" column="0">
         <widget class="ActualDemandChart" name="c_star"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRoll">
       <attribute name="title">
        <string>Roll</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelRollRateKp">
            <property name="text">
             <string>Roll Rate Kp</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBoxRollRateKp">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>-5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRollRateKi">
            <property name="text">
             <string>Roll Rate Ki</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBoxRollRateKi">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRollRateKd">
            <property name="text">
             <string>Roll Rate Kd</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBoxRollRateKd">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRollUpdate">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelRollBankKp">
            <property name="text">
             <string>Bank Kp</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBoxRollBankKp">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRollBankKi">
            <property name="text">
             <string>Bank Ki</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBoxRollBankKi">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>-0.200000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRollBankKd">
            <property name="text">
             <string>Bank Kd</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBoxRollBankKd">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QSlider" name="sliderGainRoll">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelRoll"/>
        </item>
        <item row="4" column="0">
         <widget class="ActualDemandChart" name="rollRate"/>
        </item>
        <item row="5" column="0">
         <widget class="ActualDemandChart" name="bank"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActualDemandChart</class>
   <extends>QGraphicsView</extends>
   <header>ActualDemandChart.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
